### PR TITLE
kodi-platform: fix cross-build

### PR DESCRIPTION
**Pull request by ndowens**
For now this seems to fix for atleast aarch64, until I can see if I can get kodi to build on the others

### CODE REVIEW

**Comment by ndowens** [comment]
Sorry messed up PR, tried rebasing but did not fix it

**Comment by ndowens** [comment]
According to Manual.md it says meta-package should be BSD-2-Clause so I fixed that, but new lint issue of course is no use of vlicense.

kodi-platform's fix may be sorta hacky, but it seems to work fine with cross and non-cross builds.

**Comment by ndowens** [comment]
@ericonr told you I was working on kodi-addon :) figured I'd tag you since I mentioned it to you and also to get your input.

File: srcpkgs/kodi-platform/template [hunk 1]
```diff
@@ -3,15 +3,31 @@ pkgname=kodi-platform
...
 distfiles="https://github.com/xbmc/kodi-platform/archive/${version}.tar.gz"
 checksum=39de7eb6db14de3fa33ca0e56243c3307e7598187e73ad930dabccdf068bd9b2
-nocross="kodi cmake files use absolute paths"
+
+post_build() {
+	# Remove cross-cruft
+	if [ "$CROSS_BUILD" ]; then
+		vsed -e '/libdir/s,=.*,=/usr/lib,' \
+			-e '/CXX/s,=.*,=c++,' \
+			-e '/CC/s,=.*,=gcc,' \
```
Comment (ericonr) [inline srcpkgs/kodi-platform/template:21]: I would just remove these by default...

File: srcpkgs/kodi-platform/template [hunk 2]
```diff
@@ -3,15 +3,31 @@ pkgname=kodi-platform
...
+			-e "s,${XBPS_CROSS_BASE},,g" \
+			-i build/kodiplatform.pc
+
+		vsed -e 's,/usr//,/usr/,g' \
+			-e "s,${XBPS_CROSS_BASE},,g" \
+			-e 's,/usr/usr,/usr,g' \
```
Comment (ericonr) [inline srcpkgs/kodi-platform/template:27]: Given that the file looks like

```
prefix=/usr
libdir=/usr//usr/lib
includedir=/usr/include
CXX=/usr/lib/ccache/bin/g++ 
CC=/usr/lib/ccache/bin/cc 
DEPENDENCIES=/usr/lib/libtinyxml.so

Name: kodiplatform
Description: Kodi add-on platform support library 18.0
Version: 18.0
Libs: -L -l:kodiplatform -ltinyxml
Cflags: -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe   -g -std=c++11 -fPIC
```

for native builds, the issue is that `/usr//usr/lib` appears at all. I would attribute it to `-DCMAKE_INSTALL_LIBDIR=/usr/lib`.

**Review by ericonr — COMMENTED** [review]
https://github.com/xbmc/kodi-platform/pull/30/ I'd consider carrying this patch.

**Comment by ericonr** [comment]
Fixed by @marmeladema, thanks!

**Pull request closed without merging**